### DISCUSSION TITLE
vite-plugin-ssr Vue 2 examples

### DISCUSSION THREAD

**Discussion by jmmygoggle** [vikejs/vike#533] [Discussion]
This page tags vite-plugin-ssr with Vue 2 compatibility: https://github.com/vitejs/awesome-vite#ssr
After a bit of digging I've only found Vue 3 examples so far - of which there are many in this repo alone.

Can anyone point me to example implementations using Vue 2 or an example that might be useful towards that context?

We're migrating a legacy Vue 2 app with SSR + isomorphic aspects towards compliance with new company-wide tooling that embraces Vite and Vue 3. One of our approaches is to migrate webpack (sans Vue CLI) mechanisms to Vite before tackling the Vue 3 migration.

**Comment by brillout** [comment]
I'm not aware of any Vue 2 examples. But VPS should be working just fine with Vue 2. You'll have to stitch that integration yourself though.
